package.json

```diff
@@ -69,54 +81,66 @@
     "release:minor": "npm version minor -m 'Released version %s' && npm publish",
     "release:patch": "npm version patch -m 'Released version %s' && npm publish",
     "start": "tt-serve --config config/webpack/webpack.config",
-    "start-heroku": "npm run compile:prod && tt-serve-static --port $PORT --site './build' ",
+    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
     "start-static": "npm run compile:prod && tt-serve-static --site ./build",
-    "test": "npm run lint && npm run jest",
-    "tds:generate-app-config": "node bin/generateAppConfig.js"
+    "test": "npm run lint && npm run jest && npm run wdio",
+    "tds:generate-app-config": "node bin/generateAppConfig.js",
+    "wdio": "wdio config/wdio/wdio.conf.js"
```

Comment:
I would like to remove terra-dev-site's wdio config. Previously we recommended people extend terra-dev-site's wdio configuration and it caused confusion. It would be nice if terra-dev-site also used terra-toolkit's out of the box. To do this we'd need to add a root-level wepback config to correctly resolve the site. So maybe it's not worth it replacing one config for another

Reply:
we can't because of this line:
https://github.com/cerner/terra-dev-site/blob/application-layout-updates/config/wdio/wdio.conf.js#L7

Reply:
Not a problem
![https://media.giphy.com/media/UbfdvY7bUcwkE/giphy.gif](https://media.giphy.com/media/UbfdvY7bUcwkE/giphy.gif)

Reply:
@mjhenkes I recently updated the generator and docs to create a root-level webpack.conf that is auto loaded by webpack-dev-server and wdio because I wanted to remove this, so it might not be a concern

Reply:
If we're going to do it, now is the time. I'll get this removed and update docs.